doc/userguide/output/eve/eve-json-format.rst

```diff
@@ -1804,3 +1804,111 @@ Example of HTTP2 logging, of a request and response:
       ]
     }
   }
+
+Event type: IKE
+---------------
+
+The following fields and example are for IKEv1, IKEv2 has a different output format. IKEv1 is denoted by the field version_major = 1.
```

Comment:
You can say something like

You can distinguish between IKEv1 and IKEv2 using the version_major field (which equals either 1 or 2)